drivers/storage/portworx/portworx.go

```diff
@@ -976,6 +977,29 @@ func (p *portworx) GetStorageNodes(
 
 }
 
+func (p *portworx) GetKVDBMembers(cluster *corev1.StorageCluster) (map[string]bool, error) {
```

Comment:
As discussed on the call, let's add UT's for this function. Refer examples from the links below - 
https://github.com/libopenstorage/operator/blob/master/drivers/storage/portworx/portworx_test.go#L5783
https://github.com/libopenstorage/operator/blob/master/drivers/storage/portworx/portworx_test.go#L4148


We will have to add a new mock for the PortworxServiceServer. Just add the following the `mockgen` make task -
```
$(GOPATH)/bin/mockgen -destination=pkg/mock/portworxsdk.mock.go -package=mock github.com/libopenstorage/operator/api/px PortworxServiceServer
```

Reply:
This will not work for the code I wrote as the driver function itself creates the PortworxServiceServer. Hence even if we mock the server and call the driver function still tries to initialise an actual PortworxServiceServer thus throwing and error of unknown service

Reply:
The server is running inside portworx. We cannot initialize the server in this driver function. We are just creating a connection to the server in the GetPortworxConn function. Once the grpc connection is established, we create a PortworxService client to send the RPCs.